src/lib/Liblog/chk_file_sec.c

```diff
@@ -174,32 +174,41 @@ tempstat(struct stat *sp, int isdir, int sticky, int disallow, int uid)
 	if ((~disallow & S_IWUSR) && (sp->st_uid > 10 && !(uid > 0 && uid == sp->st_uid))) {
 		/* Owner write is allowed, and UID is greater than 10 or owner is not self. */
 		rc = EPERM;
+		log_errf(rc, "tempstat", "BAD USER");
```

Comment:
Don't  say "tempstat", say \_\_func\_\_

Reply:
sorry, this is just for debugging things